server/room-battle.ts

```diff
@@ -1112,6 +1116,18 @@ export class RoomBattle extends RoomGames.RoomGame {
 		// @ts-ignore
 		this.room = null;
 	}
+	async getTeam(user: User) {
+		const id = user.id;
+		const player = this.playerTable[id];
+		if (!player) return;
+		void this.stream.write(`>requestteam ${player.slot}`);
+		const teamDataPromise = new Promise<string[]>(resolve => {
+			this.playerTeamResolver = resolve;
+		});
+		const resultStrings = await teamDataPromise;
```

Comment:
Nice! You figured out how to grab data from subprocesses! I remember you were struggling with this in the battle persistence PR.

Reply:
yeah, lol, I was actually looking at that for this